ikas-international-end-client-is-tier1-investment-bank-python-developer-with-flask-javascript-and-sql.html

```diff
@@ -0,0 +1,37 @@
+---
+title: Python Developer with Flask - Javascript and SQL
+company: Ikas International (End client is Tier1 Investment Bank)
+url: https://www.ikasinternational.com/banking-jobs/python-developer-flask-sql-javascript-greenfield-670-day/1700878
+location: London, Canary Wharf
+          # Choose one of the following options
+contract: contract
+contact:
+    name: Fadi Harb
+    email: fharb@ikasinternational.com
+    phone: 02072209344
+    # <additional contact fields>: <additional value>
+created: !!timestamp 2018-04-25
+tags:
+  - Python
```

Comment:
This seems like a great role, but can you provide more details about the skills required? For example, almost any web-based project will require some infrastructure knowledge unless it's purely an internal project. All JavaScript projects require some knowledge of JavaScript build-tooling such as NPM, Gulp, Grunt.